### PR TITLE
CSS cleanup

### PR DESCRIPTION
**Pull request by margo-fedak**
Fixed css issues.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -261,9 +261,6 @@ body {
   color: #e9e7e7;
   background-color: #232323;
   cursor: pointer;
-}
-.disabled {
-
 }
 .black-background{
```
Comment (bohdanfedak) [inline src/App.css:265]: I don't like this

**Review by bohdanfedak — CHANGES_REQUESTED** [review]
Please fix

File: src/App.css (hunk 1 above)
Comment (margo-fedak) [reply src/App.css:265]: Fixed

**Review by bohdanfedak — APPROVED** [review]
Looks good

**Pull request merged**